### DISCUSSION TITLE
Bloodlines

### DISCUSSION THREAD

**Discussion by Cronvs** [ParadoxGameConverters/ImperatorToCK3#490] [Ideas]
Have the conversion of the bloodline traits been brought up? It would be interesting to include in a ck3 campaign.

**Comment by IhateTrains** [comment]
How do you envision this? CK3 has legacies instead of bloodlines.

**Comment by Cronvs** [comment]
I envision it implemented like the prophet bloodlines in ck3, such as the sayyid or consecrated blood traits.
